src/database/gravity-db.c

```diff
@@ -837,7 +836,10 @@ char *__attribute__ ((malloc)) get_client_names_from_ids(const char *group_ids)
 	return result;
 }
 
-// Prepare statements for scanning white- and blacklist as well as gravit for one client
+// Prepare statements for scanning white- and denylist as well as gravit for one client
+
+
+// Prepare statements for scanning white- and denylist as well as gravit for one client
```

Comment:
Why is this comment appearing twice?

Also, `white-` should be renamed to `allow-` to match the other changes
```suggestion
// Prepare statements for scanning allow- and denylist as well as gravit for one client
```